lib/toxiproxy.rb

```diff
@@ -199,8 +203,12 @@ def toxics(direction)
 
   private
 
+  def self.uri
+    @uri ||= ::URI.parse('http://127.0.0.1:8474')
```

Comment:
Have a constant for the default so you can use that in the test as well (or set it to `nil`).